boards/arm/ip_k66f/ip_k66f.dts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2020 DENX Software Engineering GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_k6x.dtsi>
+
+/ {
+	model = "SEGGER MK66F IP Switch board";
+	compatible = "nxp,mk66f18", "nxp,k66f", "nxp,k6x";
+
+	aliases {
+		led0 = &red0_led;
+		led2 = &red2_led;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red0_led: led_0 {
+			gpios = <&gpioa 8 GPIO_INT_ACTIVE_HIGH>;
```

Comment:
Replace `GPIO_INT_ACTIVE_HIGH` with `GPIO_ACTIVE_HIGH`. Do this for led_2 as well

Reply:
Fixed, thanks.